flask_admin/contrib/mongoengine/filters.py

```diff
@@ -221,7 +236,21 @@ def operation(self):
         return lazy_gettext('not between')
 
 
+class ReferenceFilter(BaseMongoEngineFilter):
+
+    def clean(self, value):
+        return ObjectId(value.strip())
+
+    def apply(self, query, value):
+        flt = {'%s' % self.column.name: value}
+        return query.filter(**flt)
+
+    def operation(self):
+        return lazy_gettext('equals')
```

Comment:
How about changing this to `ObjectId equals`? Currently, it's hard to tell that it's expecting an ObjectId as input.